src/web/mjs/connectors/Mikoroku.mjs

```diff
@@ -0,0 +1,80 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+export default class Mikoroku extends Connector {
+    constructor() {
+        super();
+        super.id = 'mikoroku';
+        super.label = 'Mikoroku';
+        this.tags = [ 'manga', 'scanlation', 'indonesian' ];
+        this.url = 'https://www.mikoroku.web.id';
+        this.queryMangaTitle = 'header > h1';
+    }
+
+    async _getMangas() {
+        let mangalist = [];
+        const request = new Request(new URL('/feeds/posts/default/-/Series?orderby=published&alt=json&max-results=999', this.url), this.requestOptions);
+        const { feed } = await this.fetchJSON(request);
+        feed.entry.map(manga => {
+            const goodLink = manga.link.find(link => link.rel === 'alternate');
+            mangalist.push({
+                id: this.getRootRelativeOrAbsoluteLink(goodLink.href, request.url),
+                title: goodLink.title.trim()
+            });
+        });
+        return mangalist;
+    }
+
+    async _getChapters(manga) {
+        let uri = new URL(manga.id, this.url);
+        let request = new Request(uri, this.requestOptions);
+        let dom = (await this.fetchDOM(request, 'body'))[0];
+        let data = [...dom.querySelectorAll('script')];
+
+        const clwdrun = data.filter(el => el.text.trim().startsWith('clwd.run'));
+        let mangaid= '';
+        if (clwdrun.length > 0) {

```

Comment:
Do we really need to check for 'clwd.run' AND div#epx? 

Isn't the check for " 'clwd.run'" enough? Thats waht i was implying when i said "simplify the script"

Edit : apparently `clwd` variable contains chapters list, perhaps its easier to exploit this by injecting a script (using this.fetchUI? 

![image](https://github.com/manga-download/hakuneko/assets/10671518/8f766a3d-4ce6-4b19-a0c8-8af14c24e26b)

Reply:
done